UsacoLeeland/src/UsacoLeeland/BlockedBillboard.java

```diff
@@ -0,0 +1,114 @@
+package UsacoLeeland;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+public class BlockedBillboard 
+{
+	public static void sort(int z,int[][]x, String[]y)
+	{
+		for(int i=0; i<4; i++)
+		{
+			x[z][i]=Integer.parseInt(y[i]);
+		}
+	}

```

Comment:
This is "parse" or "read", not "sort". Naming is very important in programming, misleading names is like shooting at your foot.